examples/bert/npu/bert_snpe.json

```diff
@@ -0,0 +1,91 @@
+{
+    "input_model":{
+        "type": "PyTorchModel",
+        "config": {
+            "hf_config": {
+                "model_name": "Intel/bert-base-uncased-mrpc",
+                "task": "text-classification"
+            },
+            "io_config": {
+                "input_names": ["input_ids", "attention_mask", "token_type_ids"],
+                "input_types": ["int64", "int64", "int64"],
+                "input_shapes": [[2, 128], [2, 128], [2, 128]],
+                "output_names": ["logits"]
+            }
+        }
+    },
+    "evaluators": {
+        "snpe_evaluator": {
+            "metrics": [
+                {
+                    "name": "accuracy",
+                    "type": "accuracy",
+                    "backend": "huggingface_metrics",
+                    "data_config": "snpe_dataset",
+                    "user_config": {
+                        "inference_settings": {
+                            "snpe":{
+                                "return_numpy_results": true
+                            }
+                        }
+                    },
+                    "sub_types": [
+                        {"name": "accuracy", "priority": 1, "goal": {"type": "max-degradation", "value": 0.05}},
+                        {"name": "f1"}
+                    ]
+                }
+            ]
+        }
+    },
+    "data_configs": {
+        "snpe_dataset": {
+            "name": "snpe_dataset",
+            "type": "HuggingfaceContainer",
+            "user_script": "user_script.py",
+            "components": {
+                "post_process_data": {
+                    "type": "snpe_post_process"
+                }
+            },
+            "params_config": {
+                "model_name": "Intel/bert-base-uncased-mrpc",
```

Comment:
nt: `model_name` and `task` will be auto inserted from the input model. + `task_type` is not a config param. We use `task`

Reply:
fixed.